specification/app/resource-manager/readme.md

```diff
@@ -57,6 +57,7 @@ input-file:
   - Microsoft.App/preview/2024-10-02-preview/JavaComponents.json
   - Microsoft.App/preview/2024-10-02-preview/Jobs.json
   - Microsoft.App/preview/2024-10-02-preview/LogicAppsExtension.json
+  - Microsoft.App/preview/2024-10-02-preview/MaintenanceConfigurations.json
```

Comment:
can you name it as ManagedEnvironmentMenanceConfiguirations to align with other sub resource types like daprComponents, storages?

Reply:
Done